source/common/network/connection_impl.cc

```diff
@@ -1037,9 +1048,19 @@ void ClientConnectionImpl::connect() {
   } else {
     immediate_error_event_ = ConnectionEvent::RemoteClose;
     connecting_ = false;
+    absl::string_view remote_address_family =
```

Comment:
This seems more verbose than I think the averge operator will want in their logs.  maybe runtime guard (non-flag) and default on for E-M?

Reply:
Done.